src/diffusers/models/attention.py

```diff
@@ -158,6 +158,12 @@ def __init__(
         super().__init__()
         self.only_cross_attention = only_cross_attention
 
+        self.use_ada_layer_norm_zero = (num_embeds_ada_norm is not None) and norm_type == "ada_norm_zero"
+        self.use_ada_layer_norm = (num_embeds_ada_norm is not None) and norm_type == "ada_norm"
+        self.use_ada_layer_norm_single = norm_type == "ada_norm_single"
+        self.use_layer_norm = norm_type == "layer_norm"
+        self.use_ada_layer_norm_continuous = norm_type == "ada_norm_continuous"
```

Comment:
I am okay with this.

Reply:
Sorry for the late reply. I'd maybe just add a comment to state that they are kept for back-compatibility reasons.